seeds/data/rops.js

```diff
@@ -179,7 +180,7 @@ module.exports = [
   {
     id: 'e16a0d06-ec12-40c1-b8ce-8fce70b41992',
     projectId: '48dab955-a0b7-4ad3-bdc0-6b79d0aaeac8',
-    year: (new Date()).getFullYear(),
+    year: now.getMonth() > 6 ? now.getFullYear() : now.getFullYear() - 1,
```

Comment:
Is it worth extracting this logic into a separate function and adding a couple of tests to cover cases before and after July?

Reply:
I just realised this is test code. It is probably an overkill to extract a function and add test for test code 😄